### DISCUSSION TITLE
AWS Secretmanager config import option forcing to have JSON String as value

### DISCUSSION THREAD

**Discussion by PranobSam** [awspring/spring-cloud-aws#636] [Q&A]
Hello - 

We have a requirement where our secret is just stored a plain string and not a JSON value

Eg - /secret/demo-app/user.name-abc = "XXXXXX"

And Not as JSON shown below
/secret/demo-app/user.password-pqr  = {"password" : "XXXXXX"}

This will failed ObjectMapper wouldnt be able to work with plain string- do we have a work around flag we can use and pass property key name as parameter as well

**Comment by maciejwalkowiak** [comment]
Support for plain text secrets has been added in #338 and released in 3.0.0 M1: https://docs.awspring.io/spring-cloud-aws/docs/3.0.0-M3/reference/html/index.html#using-plain-text-secrets

There are no plans to add it to 2.4.x branch, but if you wish to contribute this change to 2.4.x I am happy to merge.

**Reply by snigdhasjg** [reply]
Oh I should have added this in my previous PR. It will be same as version `3.0.0`.

**Reply by snigdhasjg** [reply]
PR raised: #638

Please validate the doc format

**Reply by PranobSam** [reply]
Thanks @maciejwalkowiak  @snigdhasjg  - i was out and couldnt respond back. this is super cool
